### DISCUSSION TITLE
Install Nerd Fonts on Windows via Chocolatey

### DISCUSSION THREAD

**Discussion by sumo300** [ryanoasis/nerd-fonts#923] [General]
I started a project a while ago to create Chocolatey packages for each available Nerd Font within this repo and publish updates every time new font versions are released.  I wanted to contribute in some way (though my original reason for creating these packages was selfish) and would love to get feedback on how these work out.

https://github.com/sumo300/chocolatey-nerd-fonts

If you're not familiar with Chocolatey, it's a PowerShell-based tool that makes it easy to install software.  It can also be used to not only install software, but perform other tasks as well, like installing fonts.

For example, to install Cascadia Code, you'd run a simple command:

`choco install nerd-fonts-CascadiaCode`

**Comment by Finii** [comment]
Added it to the to-be-documented list: https://github.com/ryanoasis/nerd-fonts/discussions/881#discussioncomment-3488054

**Comment by Finii** [comment]
@sumo300 Just rediscovered https://github.com/ryanoasis/nerd-fonts/tree/master/chocolatey

Is this somehow related or independent?

**Reply by sumo300** [reply]
@Finii I was not aware that existed.  When searching the Chocolatey repo, there weren't any published.

**Reply by sumo300** [reply]
My repo also checks the release page regularly and will auto-generate and submit updated ones to the repo.  Feel free to take all of that scripting into this repo.  I'd just be happy for them to be published!  :)
